app-containers/containers-common/containers-common-0.56.0.ebuild

```diff
@@ -0,0 +1,110 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Common config files and docs for Containers eco-system"
+
+HOMEPAGE="https://github.com/containers/common"
+
+LICENSE="Apache-2.0"
+
+SLOT="0"
+
+DEPEND="
+	>=dev-go/go-md2man-2.0.2
+"
+
+RDEPEND="
+	app-containers/containers-image
+	app-containers/containers-storage
+	app-containers/containers-shortnames
+	net-firewall/nftables
+	net-firewall/iptables[nftables]
+	|| ( app-containers/crun app-containers/runc )
+	|| (
+		( >=app-containers/netavark-1.6.0 >=app-containers/aardvark-dns-1.6.0 )
+		>=app-containers/cni-plugins-0.9.1
+	)
+"
+
+is_live() {
+	[[ ${PV} == *9999* ]]
+}
+
+if is_live; then
```

Comment:
I don't really see the benefit here, you could just put it above with SRC_URI (as suggested above).

Look at what other ebuilds do (e.g. ntpsec-9999)